MEME/MEME/Model/MyReview/MyReviewViewController.swift

```diff
@@ -58,15 +59,17 @@ class MyReviewViewController: UIViewController, UICollectionViewDelegate, UIColl
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        ReviewListManager.shared.getReviews(portfolioId: 3, modelName: nameLabel.text ?? "", star: 5, comment: "Comment", reviewImgDtoList: [ReviewImage]()) { [weak self] result in
+        AvailableReviewManager.shared.getAvailableReview(modelId: 1, reservationId: 1, portfolioId: 1, artistNickName: "artistNickName", makeupName: "makeupName", reservationDate: "reservationDate", portfolioImg: "portfolioImg", shopLocation: "shopLocation") { [weak self] result in
```

Comment:
```suggestion
AvailableReviewManager.shared
    .getAvailableReview(
        modelId: 1,
        reservationId: 1,
        portfolioId: 1,
        artistNickName: "artistNickName",
        makeupName: "makeupName",
        reservationDate: "reservationDate",
        portfolioImg: "portfolioImg",
        shopLocation: "shopLocation"
    ) { [weak self] result in

```
과 같이 작성하면 가독성이 더 좋아질 것 같습니다!